include/ignition/common/Time.hh

```diff
@@ -105,6 +113,15 @@ namespace ignition
       /// On Windows the return value is always common::Time::Zero.
       public: static Time Sleep(const common::Time &_time);
 
+      public: static void GetSecondsAndNanoseconds(
```

Comment:
This looks similar to [ignition::math::durationToSecNsec](https://github.com/ignitionrobotics/ign-math/blob/6a3c4a32b18e6bd7956fecfe53ccfe14bfcd2907/include/ignition/math/Helpers.hh#L573-L584). How about putting the new functions there instead of `ign-common`?

Reply:
Would that be an issue for people that were previously depending on ign-common for time-related functions?  Or would both generally be available everywhere

Reply:
Since `ign-math` is a required dependency of `ign-common3` anyway, I don't think current users should need to add any new dependencies.